src/breakpoints/body.tsx

```diff
@@ -65,13 +65,15 @@ const BreakpointsComponent = ({ model }: { model: BreakpointsModel }) => {
 
   return (
     <>
-      {breakpoints.map(entry => (
-        <BreakpointCellComponent
-          key={entry[0]}
-          breakpoints={entry[1]}
-          model={model}
-        />
-      ))}
+      {breakpoints
+        .filter(find => find[0] === model.currentBreakpointsId)
```

Comment:
Does this mean that we display the breakpoints for the currently selected cell only?

Reply:
Yes

Reply:
Or actually the last cell that was updated.

Reply:
I think we still want to show the breakpoints for the other cells. To be able to select the cell where the breakpoint is defined when they are clicked.

Reply:
![Peek 2020-04-24 12-58](https://user-images.githubusercontent.com/45068804/80205762-7867b680-862b-11ea-9bf7-8079961592ad.gif)

How looks now.

Reply:
![Peek 2020-04-24 13-01](https://user-images.githubusercontent.com/45068804/80205939-cb416e00-862b-11ea-8b40-75ca05daa956.gif)

How works now

Reply:
When adding the breakpoint to the second cell the breakpoint list should show 3 breakpoints:

- 2 for the first cell
- 1 for the second cell

Reply:
Sure, I'll figure something out